autopilot_bindings/lib.rs

```diff
@@ -289,11 +289,16 @@ fn op_click(_interface: &mut dyn Interface, zero_copy: &mut [ZeroCopyBuf]) -> Op
 fn op_scroll(_interface: &mut dyn Interface, zero_copy: &mut [ZeroCopyBuf]) -> Op {
     let data = &zero_copy[0][..];
     // convert arg to string
-    let _data_str = std::str::from_utf8(&data[..]).unwrap().to_string();
+    let data_str = std::str::from_utf8(&data[..]).unwrap().to_string();
     let fut = async move {
         let (tx, _rx) = futures::channel::oneshot::channel::<Result<(), ()>>();
         std::thread::spawn(move || {
-            rs_lib::mouse::scroll(rs_lib::mouse::ScrollDirection::Up, 5 as u32);
+            if data_str == "up" {
+                rs_lib::mouse::scroll(rs_lib::mouse::ScrollDirection::Up, 5 as u32);
+            } 
+            if data_str == "down" {
+                rs_lib::mouse::scroll(rs_lib::mouse::ScrollDirection::Down, 5 as u32);
+            }
```

Comment:
Consider using `match` instead of `if`'s
```rust
match data_str {
  "up" => rs_lib::mouse::scroll(rs_lib::mouse::ScrollDirection::Up, 5 as u32),
  ...
}
```

Reply:
updated